doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,112 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- New CMORizers have been added:
```

Comment:
Maybe mention all the datasets that have new CMORizers here instead of just two?

Reply:
Done in 9b506a25e (and also includes better formatting more like what was done in the v2.10.0 release notes) 👍